dcmgr/lib/dcmgr/endpoints/12.03/responses/instance.rb

```diff
@@ -124,4 +123,20 @@ def generate
       @instance.monitor_items
     end
   end
+
+  class InstancePassword < Dcmgr::Endpoints::ResponseGenerator
+    def initialize(instance)
+      raise ArgumentError if !instance.is_a?(Dcmgr::Models::Instance)
```

Comment:
Don't raise argument errors without a message. If this error gets raised, it will show ArgumentError in the log with no further information and you would have to go read the source code to know what the problem was.